scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -33,11 +33,22 @@ task GetToolVersions {
     set -o errexit -o nounset -o pipefail -o xtrace
 
     # Scrape out various workflow / workspace info from the localization and delocalization scripts.
-    sed -n -E 's!.*gs://fc-(secure-)?([^\/]+).*!\2!p' /cromwell_root/gcs_delocalization.sh | sort -u > ~{workspace_id_output}
-    sed -n -E 's!.*(gs://(fc-(secure-)?[^\/]+)).*!\1!p' /cromwell_root/gcs_delocalization.sh | sort -u > ~{workspace_bucket_output}
-    sed -n -E 's!.*gs://fc-(secure-)?([^\/]+)/submissions/([^\/]+).*!\3!p' /cromwell_root/gcs_delocalization.sh | sort -u > ~{submission_id_output}
-    sed -n -E 's!.*gs://fc-(secure-)?([^\/]+)/submissions/([^\/]+)/([^\/]+)/([^\/]+).*!\5!p' /cromwell_root/gcs_delocalization.sh | sort -u > ~{workflow_id_output}
-    sed -n -E 's!.*(terra-[0-9a-f]+).*# project to use if requester pays$!\1!p' /cromwell_root/gcs_localization.sh | sort -u > ~{google_project_output}
+    if [[ -e /cromwell_root/gcs_delocalization.sh ]]
+    then
+      CROMWELL_ROOT=/cromwell_root
+    elif [[ -e /mnt/disks/cromwell_root/gcs_delocalization.sh ]]
+    then
+      CROMWELL_ROOT=/mnt/disks/cromwell_root
```

Comment:
I know that AoU uses individual instances of Cromwell instead of Terra's central instance. Does this assignment account for this?

Reply:
This code should be fine to run on GCP Batch or PAPI regardless of where the Cromwell instance is deployed.

But you will need to update the configs for your Cromwells to point to GCP Batch. Did the Cromwell team help you with this before?

Reply:
And also upgrade your version of Cromwell to get all the GCP Batch fixes...

Reply:
They may have, but this is the first time that I have dealt with this, so I will look into it.

Thanks for your help with all this!